src/renderer/lib/state.js

```diff
@@ -69,7 +70,6 @@ function load (cb) {
     if (err) {
       console.log('Missing config file: Creating new one')
       onSavedState(null, {
-        locale: null,
```

Comment:
Lets use sane defaults, `null` isn't one of them.

Reply:
Note that this value can be moved up to the default values above, since we're merely doing `Object.assign()` on top of the default values, this value might as well be part of the default values.